### PR TITLE
feat #5 created Email function and tested in EmailTest.

### PR DESCRIPTION
**Pull request by ElisabethChen**

### CODE REVIEW

File: ci/src/main/java/ci/Email.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
+package ci;
+
+import java.util.Properties;
+
+import javax.mail.*;
+import javax.mail.internet.*;
```
Comment (cadye) [inline ci/src/main/java/ci/Email.java:6]: javax.mail.* should include javax.mail.internet.*, so the second import statement is redundant

File: ci/src/main/java/ci/Email.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    /**
+     * create the massage
+     * 
+     * @param content the content of the message
+     * @throws AddressException
+     * @throws MessagingException
+     */
+    private void createMesssage(String content) throws AddressException, MessagingException {
```
Comment (cadye) [inline ci/src/main/java/ci/Email.java]: there are 3 's' in "message".

File: ci/src/main/java/ci/Email.java (hunk 1 above)
Comment (ElisabethChen) [reply ci/src/main/java/ci/Email.java:6]: some functions can't be resovled if I do not import mail.internet. E.g.:
image

File: ci/src/main/java/ci/Email.java (hunk 2 above)
Comment (ElisabethChen) [reply ci/src/main/java/ci/Email.java]: ups..., true. I will change it!

**Review by cadye — APPROVED** [review]
good stuff

**Pull request merged**